blocks/editable/index.js

```diff
@@ -273,6 +285,10 @@ export default class Editable extends wp.element.Component {
 		const focusPosition = this.getRelativePosition( element );
 		const bookmark = this.editor.selection.getBookmark( 2, true );
 		this.setState( { alignment, bookmark, formats, focusPosition } );
+
+		if ( this.props.onNodeChange ) {
```

Comment:
We can safely remove this, as in your implementation we've exposed the `editor` instance via `onSetup` and bound to the event directly in the block.